packages/desktop/src/renderer/index.tsx

```diff
@@ -20,7 +17,7 @@ import FileViewerServiceElectron from "../services/FileViewerServiceElectron";
 import PersistentConfigServiceElectron from "../services/PersistentConfigServiceElectron";
 import NotificationServiceElectron from "../services/NotificationServiceElectron";
 import { GlobalVariableChannels } from "../util/constants";
-
+import { useKeyDown } from "../../../core/hooks/useKeyDown/useKeyDown"; // Import the useKeyDown hook
```

Comment:
I _think_ `hooks` contains exports for other hooks so it would be cool to simplify this import like this if so
```suggestion
import { useKeyDown } from "../../../core/hooks";
```

Reply:
I didn't see an export location in hooks. For the other hooks like useHelpOptions it looks like we import directly.